.github/bf2-arch-bot.yml

```diff
@@ -0,0 +1,13 @@
+# Github login name of the bot itself
+botUserLogin: bf2-arch-bot
+
+# The time, in hours, to wait between checking for stalled discussions
+# 1440 minutes = 24 hours
+stalledDiscussionPollTimeMins: 1440
+
+# Github logins of people who can do "/create adr" etc on issues
+recordCreationApprovers:
+- emmanuelbernard
+- tombentley
```

Comment:
I don't know who should be on this list...I just copied Tom's example config and added Emmanuel.